ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -124,44 +136,88 @@ class TreegraphSeries extends OrganizationSeries {
              */
             marker: {
-                style: {
-                    cursor: 'pointer'
-                },
                 radius: 10,
                 lineWidth: 0,
                 symbol: 'circle',
                 fillOpacity: 1,
                 states: {}
             },
+            link: {
+                /**
+                 * The color of the links between nodes.
+                 *
+                 * @type {Highcharts.ColorString}
+                 * @private
+                 */
+                color: Palette.neutralColor60,
+                /**
+                 * The line width of the links connecting nodes, in pixels.
+                 * @type {number}
+                 *
+                 * @private
+                 */
+                lineWidth: 1,
+                /**
+                 * Radius for the rounded corners of the links between nodes.
+                 * Works for `default` link type.
+                 *
+                 * @private
+                 */
+                radius: 10,
+                cursor: 'default',
+                /**
+                 * Type of the link shape.
+                 *
+                 * @sample   highcharts/series-treegraph/link-types
+                 *           Different link types
+                 *
+                 * @type {'default' | 'curved' | 'straight'}
+                 * @product highcharts
+                 *
+                 */
+                type: 'curved'
+                /**
+                 * Modifier of the shape of the curved link. Works best for
+                 * values between 0 and 1, where 0 is a straight line, and 1 is
+                 * a shape close to the default one.
+                 *
+                 * @default 0.5
+                 * @type {number}
+                 * @since next
+                 * @product highcharts
+                 * @apioption series.treegraph.link.offset
+                 *
+                 */
+            },
             /**
              * @extends plotOptions.series.tooltip
-             * @exclude pointFormat, pointFormatter
              */
             tooltip: {
                 /**
                  * The HTML of the point's line in the tooltip. Variables are
-                 * enclosed by curly brackets. Available variables are point.x,
-                 * point.y, series.name and series.color and other properties on
-                 * the same form. Furthermore, point.y can be extended by the
-                 * tooltip.valuePrefix and tooltip.valueSuffix variables. This
-                 * can also be overridden for each series, which makes it a good
-                 * hook for displaying units.In styled mode, the dot is
-                 * colored by a class name rather than the point color.
+                 * enclosed by curly brackets. Available variables are
+                 * `point.x`,  `point.y`, `series.name` and `series.color` and
```

Comment:
Do we use `point.y` in the tooltip at all? I am asking because in another place about the data structure it is mentioned that we don't require any values. Maybe we may create this info more about our specific series?